unabridged-works-volume-2/original/29-letters.adoc

```diff
@@ -91,17 +97,18 @@ thine eyes may be opened to see the way of righteous government in the true ligh
 From one who mourns over the misery of mankind,
 longing for the redemption of those that go astray, and a true lover of thy soul.
 
+[.signed-section-signature]
 I+++.+++ P.
 
-Aylesbury prison, where I am visiting some of my dear
-
-Friends in God`'s eternal truth, 17th 7th mo.
+[.signed-section-context-close]
+Aylesbury prison, where I am visiting some of my dear Friends in God`'s eternal truth,
+17th 7th mo.
 1661
 
 [.postscript]
 ====
 
-Postscript.--Let thy government be like unto God`'s: even a yoke to the unjust,
+Postscript. Let thy government be like unto God`'s: even a yoke to the unjust,
```

Comment:
This is sort of an archaic combination of punctuation, but I don't consider it an error (in fact, it's the punctuation that the linter recommends when a postcript starts with "Postscript". You don't need to change this one, but probably going forward you can leave these alone. 👍